test/test_prims.py

```diff
@@ -18,63 +18,68 @@ class TestPrims(TestCase):
     @onlyCUDA
     @dtypes(torch.float32)
     def test_broadcast_in_dim(self, device, dtype):
-        def _wrapper(a, shape, broadcast_dimensions):
-            return prims.broadcast_in_dim(a, shape, broadcast_dimensions)
+        # nvfuser is not currently capable of realizing a broadcasted tensor
+        # when the broadcast is the only operation.  Another op is needed.
+        def _wrapper(a, b, broadcast_dimensions):
+            a_bc = prims.broadcast_in_dim(a, b.shape, broadcast_dimensions)
+            return prims.add(a_bc, b)
 
         traced = make_traced(_wrapper)
         make_arg = partial(make_tensor, device=device, dtype=dtype)
 
-        # TODO: FIXME:
-        # for executor in ('aten', 'nvfuser'):
-        for executor in ("aten",):
+        for executor in ('aten', 'nvfuser'):
             fn = partial(traced, executor=executor)
             # Same shape
             shape = (5, 5)
             a = make_arg(shape)
-            result = fn(a, shape, (0, 1))
+            b = make_arg(shape, low=0.0, high=0.0)
```

Comment:
Why make b a tensor of just zeros?

Reply:
Same question here, wasn't this going to hide the indexing issue into b (if there's any).

Is this just to make test easier?

Reply:
I made the `b` Tensor full of zeros because I am really testing the `broadcast_in_dim` operation but I need an `add` or any other operation after the `broadcast_in_dim` to realize the broadcasted output.  Using zeros made it easier to figure out whether the broadcast was successful in moving the values to the right spots without polluting them non-zero value.